src/utils/DiscordUtils.ts

```diff
@@ -13,16 +13,16 @@ class DiscordUtils {
 			}
 		}
 
-		// Username + discriminator
+		// Username and/or discriminator
 		if ((/^.*#[0-9]{4}$/g).test(value)) {
 			const [username, discriminator] = value.split("#");
-			const memberList = await guild.members.fetch({query: username});
+			const memberList = await guild.members.fetch({query: username, limit: 1000000000});
```

Comment:
I feel like there must be a performance loss queuing based on a limit this large. Is this just a number picked randomly? I think we could reduce it significantly.